package.json

```diff
@@ -36,8 +37,8 @@
     "fetch-icon": "./scripts/fetch-icon.js",
     "resize-images": "./scripts/resize-images.js",
     "purgecss": "./scripts/purgecss.js",
-    "start": "npm run purgecss; bundle exec jekyll serve",
-    "build": "npm run purgecss; bundle exec jekyll build"
+    "start": "npm run purgecss; eleventy --serve",
+    "build": "npm run purgecss; eleventy"
```

Comment:
I know you inherited this, but I'm curious why we used `;` instead of `&&` 🤔

Reply:
Maybe speed? 🤷‍♀️ I changed it.